src/Microsoft.AspNet.Razor/Generator/Compiler/CodeBuilder/CSharp/CSharpTagHelperCodeRenderer.cs

```diff
@@ -440,7 +440,7 @@ private void RenderEndTagHelpersScope()
                                                   _tagHelperContext.ScopeManagerEndMethodName);
         }
 
-        private void RenderTagOutput(string tagOutputMethodName)
+        private void RenderTagOutput(string tagOutputMethodName, bool isString = false)
```

Comment:
This flag is to signify if its a `string` or `ITextWriterCopyable`. Perhaps, I can call it `isTagHelperOutputString`?